src/main/java/com/openlattice/shuttle/dates/JavaDateTimeHelper.java

```diff
@@ -16,138 +16,138 @@
-            try {
-                LocalDateTime ldt = LocalDateTime.parse( date, formatter );
-                //                if (ldt.isBefore(LocalDateTime.of(1900, 1, 1, 0, 0, 0))){return null;}
-                return ldt.atZone( tz.toZoneId() ).toOffsetDateTime();
-            } catch ( Exception e ) {
-                if ( i == datePatterns.length - 1 ) {
-                    logger.error( "Unable to parse datetime {}, please see debug log for additional information.",
-                            date );
-                }
-            }
         }
-        return null;
+        return ldt.atZone(tz.toZoneId()).toLocalDate();
     }
 
-    public OffsetDateTime parseDateAsDateTime( String date ) {
-        if ( shouldIgnoreValue( date ) )
+    public LocalTime parseDateTimeAsTime(String datetime) {
+        if (shouldIgnoreValue(datetime))
             return null;
-        for ( int i = 0; i < datePatterns.length; ++i ) {
-            DateTimeFormatter formatter = formatters.get( i );
-            try {
-                LocalDateTime ldt = LocalDate.parse( date, formatter ).atTime( 0, 0 );
+        LocalDateTime ldt = parseDateTime(datetime).toLocalDateTime();
+        if (ldt == null) {
+            return null;
+        }
+        return ldt.atZone(tz.toZoneId()).toLocalTime();
+    }
 
-                return ldt.atZone( tz.toZoneId() ).toOffsetDateTime();
-            } catch ( Exception e ) {
-                if ( i == datePatterns.length - 1 ) {
-                    logger.error( "Unable to parse datetime {}, please see debug log for additional information.",
-                            date );
-                }
-            }
+    public OffsetDateTime parseDateAsDateTime(String date) {
+        LocalDate ld = parseDate(date);
+        if (ld == null) {
+            return null;
         }
-        return null;
+        LocalDateTime ldt = ld.atTime(0, 0);
+        return ldt.atZone(tz.toZoneId()).toOffsetDateTime();
     }
 
-    public LocalTime parseTime( String time ) {
-        if ( shouldIgnoreValue( time ) )
+    public LocalTime parseTime(String time) {
+
+        if (shouldIgnoreValue(time))
             return null;
-        for ( int i = 0; i < datePatterns.length; ++i ) {
-            DateTimeFormatter formatter = formatters.get( i );
+        for (int i = 0; i < datePatterns.length; ++i) {
+            DateTimeFormatter formatter = formatters.get(i);
             try {
-                return LocalTime.parse( time, formatter );
-            } catch ( Exception e ) {
-                if ( i == datePatterns.length - 1 ) {
-                    logger.error( "Unable to parse time {}, please see debug log for additional information.", time );
+                return LocalTime.parse(time, formatter);
+            } catch (Exception e) {
+                if (i == datePatterns.length - 1) {
+                    logger.error("Unable to parse time {}, please see debug log for additional information.", time);
```

Comment:
If you are going to catch the exception, please include it as an argument in the logger so that the exception information is logged.